src/main/java/racing/domain/Participants.java

```diff
@@ -0,0 +1,48 @@
+package racing.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+public class Participants {
+  private List<Car> cars;
+
+  public Participants(List<CarName> carNames){
+    this.cars = initCars(carNames);
+  }
+
```

Comment:
```suggestion
     Participants(List<Car> cars)
```
주생성자는 기본으로 구현할 것을 추천
객체의 필드 값은 주생성자만 초기화할 수 있도록 구현하는 습관을 들인다.